ui/src/pages/frontsite/Documents/DocumentsDetails/LoanRequestForm/LoanRequestForm.js

```diff
@@ -30,6 +29,8 @@ export default class LoanRequestForm extends Component {
     this.state['deliveryMethod'] = this.withDeliveryMethod
       ? this.deliveryMethods[0].value
       : null;
+
+    this.props.initializeState();
```

Comment:
in general the state is initialised by redux, was there any reason to initialise here?

Reply:
And I would say that it should happen to `componentDidMount`, maybe there is a reason for this here.

Reply:
Because if I remove it from there, when I go to a document make a loan request and an error/success message appears, if I go back and forward again in the browser the message is still there.